applications/MappingApplication/custom_mappers/beam_mapper.h

```diff
@@ -0,0 +1,603 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Juan Ignacio Camarotti, Philipp Bucher
```

Comment:
I would put the names in two different lines

Reply:
@juancamarotti I would argue to only put Gaby as author here, what do you think?

```suggestion
//  Main authors:    E. G. Loera Villeda
```

Reply:
you could add yourself as contributor?

Reply:
Sure! You are completely right, I will put myself as a contributor.